miso-web/src/main/webapp/scripts/hot_boxable.js

```diff
@@ -69,19 +69,21 @@ HotTarget.boxable = (function() {
     for ( var key in freeByAlias) {
       freeByAlias[key].sort(sortFunction);
     }
+    var changes = [];
     for (var row = 0; row < rowCount; row++) {
       var boxAlias = table.getDataAtRowProp(row, 'boxAlias');
       if (boxAlias && !table.getDataAtRowProp(row, 'boxPosition')) {
         var free = freeByAlias[boxAlias];
         if (free && free.length > 0) {
           var pos = free.shift();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "pos" as this name is already used in declaration at line 61. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS2814)